### DISCUSSION TITLE
Save Feishin settings in persistent storage on Docker

### DISCUSSION THREAD

**Discussion by zpaolo11x** [jeffvli/feishin#757] [Ideas]
I run Feishin through Docker with a Navidrome server and it's gorgeous. The only issue I have is that since settings are saved in browser local storage, they are lost if I access from a different browser. Would it be possible to add an option to save settings in a persistent volume on docker?

**Comment by Anexgohan** [comment]
Yes please this +1

**Comment by Qwerty-Space** [comment]
Has this been addressed by a dev at all?
